2.1-segwit-version-1.ipynb

```diff
@@ -258,20 +256,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Spending a segwit v1 output with a MuSig public key\n",
+    "## Part 2 (Case Study): 2-of-2 multisig\n",
+    "\n",
+    "Alice stores her bitcoin using a combination of an offline hardware wallet and online wallet. She currently uses P2WSH 2-of-2 multisig, which has some drawbacks:\n",
     "\n",
-    "In this exercise, we'll use the Bitcoin Core wallet to create a transaction that sends to a 2-of-2 musig aggregate pubkey (segwit v1 address).\n",
+    "- spending a P2WSH multisig output is more expensive than spending a single signature P2WPKH output, since multiple pubkeys and signatures need to be included in the witness\n",
+    "- spending from the P2WSH output reveals that the coins were encumbered using a multisig setup. Anyone who transacted with Alice (paid or was paid by) can see this easily, and even entities who do not transact directly with Alice can discover this with some chain analysis. Revealing her wallet setup may be bad for Alice's privacy and safety.\n",
     "\n",
-    "We'll then manually create a transaction that spends the MuSig output and sends an output back to the Bitcoin Core wallet. We'll use Bitcoin Core's `testmempoolaccept()` RPC method to verify that the transaction is valid."
+    "In this chapter, we'll show how Alice can change to using a MuSig aggregated public key, saving her transaction fees and protecting her privacy."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 2.1.7 _Programming Exercise:_ Generate segwit v1 addresses for a 2-of-2 MuSig aggregate pubkey\n",
+    "### Spending a segwit v1 P2SH 2-of-2 multisig\n",
```

Comment:
I think segwit v1 P2SH should be segwit v0 P2WSH

Reply:
Good catch. Fixed!